hepcrawl/loaders.py

```diff
@@ -165,5 +164,40 @@ class HEPLoader(ItemLoader):
     )
     urls_out = ListToValueDict()
 
+    def get_dois(self, dois=None, material=None):
+        """Adds `dois` field to `self`.
+
+        Args:
+            dois (list):The `dois` represented by a list of strings.
+            material (str):The `material` of the dois.
+
+        Examples:
+            Using `dois`::
+
+                >>> from hepcrawl.items import HEPRecord
+                >>> record = HEPLoader(item=HEPRecord(), selector=selector)
+                >>> record.get_dois(dois=dois_list_str)
+
+            Using `dois` and `material`::
+
+                >>> from hepcrawl.items import HEPRecord
+                >>> record = HEPLoader(item=HEPRecord(), selector=selector)
+                >>> record.get_dois(dois=dois_list_str, material='preprint')
+        """
+        if dois is None or []:
```

Comment:
You can simplify this with:
```python
if not dois:
```
Unless empty string, False and 0 are some cases you want to use (I don't think so).